kolibri/plugins/learn/assets/src/composables/useDownloadRequests.js

```diff
@@ -23,111 +27,63 @@ const downloadRequestsTranslator = createTranslator('DownloadRequests', {
 });
 
 // The reactive is defined in the outer scope so it can be used as a shared store
-const downloadRequestMap = reactive({
-  downloads: {},
-  totalPageNumber: 0,
-  totalDownloads: 0,
-});
+const downloadRequestMap = reactive({});
+const totalStorage = ref(0);
+const loading = ref(true);
 
 export default function useDownloadRequests(store) {
   store = store || getCurrentInstance().proxy.$store;
   function fetchUserDownloadRequests(params) {
-    const { page, pageSize } = params;
-    const loading = ref(true);
-    const dummyDownloadRequests = [
-      {
-        id: '2ea9bda8703241be89b5b9fd87f88815',
-        user_id: store.getters.currentUserId,
-        reason: 'USER_INITIATED',
-        facility_id: store.getters.userFacilityId,
-        status: 'QUEUED',
-        date_added: new Date(),
-        resource_metadata: {
-          title: 'Intro to addition',
-          file_size: 1113580,
-          learning_activities: ['UD5UGM0z'],
-        },
-        node_id: '2ea9bda8703241be89b5b9fd87f88815',
-      },
-      {
-        id: '9e53d545aaf44c3787a29a34b189c56a',
-        user_id: store.getters.currentUserId,
-        reason: 'USER_INITIATED',
-        facility_id: store.getters.userFacilityId,
-        status: 'QUEUED',
-        date_added: new Date(),
-        resource_metadata: {
-          title: 'PDF 1 page',
-          file_size: 3113580,
-          learning_activities: ['wA01urpi'],
-        },
-        node_id: '9e53d545aaf44c3787a29a34b189c56a',
-      },
-    ];
-    setTimeout(() => {
-      set(downloadRequestMap, 'downloads', {});
-      for (let i = 0; i < pageSize; i++) {
-        const index = (page - 1) * pageSize + i;
-        if (index >= dummyDownloadRequests.length) {
-          break;
-        }
-        set(
-          downloadRequestMap.downloads,
-          dummyDownloadRequests[index].node_id,
-          dummyDownloadRequests[index]
-        );
-      }
-      set(
-        downloadRequestMap,
-        'totalPageNumber',
-        Math.ceil(dummyDownloadRequests.length / pageSize)
-      );
-      set(downloadRequestMap, 'totalDownloads', dummyDownloadRequests.length);
-      set(loading, false);
-    }, 500);
-    return loading;
+    const storage = 0;
+    return ContentRequestResource.list(params)
+      .then(downloadRequests => {
+        set(downloadRequestMap, downloadRequests);
+        set(totalStorage, storage);
```

Comment:
It looks like the `totalStorage` value is always being set to `0` - it starts at `0`, and here is being set to `0` again. This seems like it might not be right.